doc/dev/on-boarding.md

```diff
@@ -0,0 +1,143 @@
+# On-boarding
+
+This page provides learning resources for new GREASE developers.
+
+## Binary analysis
+
+### General background
+
+- [GDB in The Architecture of Open-Source Applications](https://aosabook.org/en/v2/gdb.html)
+- [Phrack](https://phrack.org/)
+- [Eli Bendersky's website](https://eli.thegreenplace.net/)
+  - [Posts on debuggers](https://eli.thegreenplace.net/tag/debuggers)
+  - [Posts on assembly](https://eli.thegreenplace.net/tag/assembly)
+    - [Load-time relocation of shared libraries](https://eli.thegreenplace.net/2011/08/25/load-time-relocation-of-shared-libraries)
+    - [Position Independent Code (PIC) in shared libraries](https://eli.thegreenplace.net/2011/11/03/position-independent-code-pic-in-shared-libraries)
+    - [Stack frame layout on x86-64](https://eli.thegreenplace.net/2011/09/06/stack-frame-layout-on-x86-64)
+- [Binary symbolic execution with KLEE-Native](https://blog.trailofbits.com/2019/08/30/binary-symbolic-execution-with-klee-native/)
+
+### Try it out!
+
+- [ ] Use bintools
+  - [ ] Read `objdump` tldr, disassemble something!
+  - [ ] Read `readelf` tldr, examine an ELF file!
+  - [ ] Read `nm` tldr, look at some symbols!
```

Comment:
Read what, specifically? The source code of the tools? The output of the tools? Something else?